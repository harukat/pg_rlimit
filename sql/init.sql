CREATE EXTENSION pg_rlimit;
\! ulimit -v
\echo precondition: ulimit -v is unlimited(-1)
