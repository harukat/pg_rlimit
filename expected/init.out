CREATE EXTENSION pg_rlimit;
\! ulimit -v
unlimited
\echo precondition: ulimit -v is unlimited(-1)
precondition: ulimit -v is unlimited(-1)
